lib/phoenix_live_view/test/client_proxy.ex

```diff
@@ -778,11 +778,11 @@ defmodule Phoenix.LiveViewTest.ClientProxy do
     {:ok, root}
   end
 
-  defp maybe_cid(_tree, nil) do
+  def __maybe_cid__(_tree, nil) do
```

Comment:
What is the rational behind enclosing the function name with `__`, there are no other function in the module using this naming convention?

Reply:
Correct me if I'm wrong but I think it signals like "even though this is a public function, consider it private". In this case it's being called from `live_view_test.ex`.